README.md

```diff
@@ -122,21 +114,21 @@ for the `crates-io` public crates.
    removing the inner whitespace from the problematic keys should work around
    this issue.
 
-   ```
+   ```text
    error: while parsing a TOML string at /nix/store/.../overlay/mkcrate.nix:31:14: Bare key 'cfg(target_os = "linux")' cannot contain whitespace at line 45
    ```
 
 ## Design
 
 This Nixpkgs overlay builds your Rust crates and binaries by first pulling the
 dependencies apart, building them individually as separate Nix derivations and
-linking them together. This is achieved by basically passing linker flags to the
-`cargo` invocations and the underlying `rustc`/`rustdoc` invocations.
+linking them together. This is achieved by passing custom linker flags to the
+`cargo` invocations and the underlying `rustc` and `rustdoc` invocations.
 
-The overlay is built on top of the work by James Kay. In addition, this overlay
```

Comment:
Why the James Kay mention is removed?

Reply:
As mentioned in the commit message for c2c03f1, the same credit is repeated in the _Credits_ section immediately below.